src/libraries/System.IO.Compression.Brotli/src/System/IO/Compression/enc/BrotliCompressionOptions.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.IO.Compression
+{
+    /// <summary>
+    /// Provides compression options to be used with <see cref="BrotliStream"/>.
+    /// </summary>
+    public sealed class BrotliCompressionOptions
+    {
+        private int _quality = BrotliUtils.Quality_Default;
+
+        /// <summary>
+        /// Gets or sets the compression quality for a Brotli compression stream.
+        /// </summary>
+        /// <exception cref="ArgumentOutOfRangeException" accessor="set">Thrown when the value is less than 0 or greater than 11.</exception>
+        /// <remarks>
+        /// The higher the quality, the slower the compression. Range is from 0 to 11.
```

Comment:
Since it seems 0 is the default, let's indicate it in the docs:
```suggestion
        /// The higher the quality, the slower the compression. Range is from 0 to 11. The default value is 0.
```

Reply:
Default is 4

Reply:
Ah okay. There was no answer in Stephen's comment so I assumed you were agreeing with (Y) that it was 0. Anyway, thanks for adding the 4.